packages/flutter_tools/lib/src/web/file_generators/flutter_js.dart

```diff
@@ -2,374 +2,20 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:path/path.dart' as path;
+
+import '../../globals.dart' as globals;
+import 'helper.dart';
 
 /// Generates the flutter.js file.
 ///
 /// flutter.js should be completely static, so **do not use any parameter or
 /// environment variable to generate this file**.
 String generateFlutterJsFile() {
```

Comment:
Do we even need this method anymore? Can we just copy `flutter.js` to the correct build destination as needed?

Reply:
This complicates things in tests because inside a test, the file system doesn't match reality.